tests/contrib/operators/postgres_local_executor.cfg

```diff
@@ -26,3 +26,4 @@
 [core]
 executor = LocalExecutor
 sql_alchemy_conn = postgresql:///airflow/airflow.db
+fernet_key = 4pB3MZt2Y-CPROzonpso4wmf-e_GOrG1FMf97m54uVg=
```

Comment:
Should this be committed?

Reply:
Please review only latest commit. We have other things on this branch, but this is necessary for development purposes.

Reply:
Yep, sorry. My bad.